doc/development_process/release_process.rst

```diff
@@ -98,6 +99,25 @@ Here is the description of the various moderation levels:
   - Bug Fixes: P1 and P2
   - Documentation + Test Coverage
 
+Release Quality Metrics
```

Comment:
Maybe 'Quality release criteria (QRC)'?

Reply:
> Quality release criteria (QRC)'

That sounds like we have a quality release, wouldn't it better to say "Release Quality Criteria", so a release has a quality criteria (and not the other way around). This is probably just semantics :)

Reply:
I have no strong preference here. I've been calling them quality metrics for a while and hence the language but I concede that we don't seem to have a common language here. But if we want to change my language I would lean towards @jukkar suggested phrasing.

Reply:
I've gone with @jukkar suggested wording